ferretdb_packaging/definedockertags/main.go

```diff
@@ -0,0 +1,285 @@
+// Copyright 2021 FerretDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package main contains tool for defining Docker image tags on CI.
+package main
```

Comment:
This is very much similar to https://github.com/FerretDB/FerretDB/blob/main/tools/definedockertag/main.go but not quite close enough because it builds tags prefixed with pg version. See tests.